servers/services/src/main/java/org/projectnessie/services/rest/BaseResource.java

```diff
@@ -60,10 +61,6 @@ protected BaseResource(
     this.accessChecker = accessChecker;
   }
 
-  Optional<Hash> getHash(String ref) {
```

Comment:
were these methods not used? Seems like we just added them no?

Reply:
Nope, no longer used.